secboot/encrypt.go

```diff
@@ -74,3 +79,33 @@ func (key RecoveryKey) Save(filename string) error {
 	}
 	return osutil.AtomicWriteFile(filename, key[:], 0600, 0)
 }
+
+// XXX: copied from github.com/snapcore/secboot/crypto.go:RecoveryKey.String()
```

Comment:
we should avoid copying this I think.

Reply:
Hm, I wonder what the best way to avoid this is. I could import it in encrypt_tpm.go and provide a dummy encrypt_dummy.go implementation but then the "nosecboot" tests will be tricky because the "String()" output is part of the api test output that is checked in apiSuite.TestSystemGetRecoveryKeyAsRootHappy. Ideas welcome!

Reply:
just add skip code if the result of RecoverKey(something).String() is the dummy version?